R-package/src/Makevars.in

```diff
@@ -26,6 +26,7 @@ OBJECTS = \
     boosting/gbdt_model_text.o \
     boosting/gbdt_prediction.o \
     boosting/prediction_early_stop.o \
+    boosting/mvs.o\
```

Comment:
```suggestion
    boosting/mvs.o \
```

Could you please  add a space before the `\` here? Just to make this a bit easier to read.

Reply:
Accepted in d499d15